src/javascripts/search.js

```diff
@@ -0,0 +1,58 @@
+UK_Parliament.search = function () {
+  var
+    header = document.querySelector('header'),
+    logo = header.querySelector('.uk_parliament'),
+    searchComponent = header.querySelector('.search__global'),
+    input = header.querySelector('input'),
+    button = header.querySelector('button'),
+    searchToggle = header.querySelector('.search__global__toggle');
+
+  if(searchComponent) {
+    checkScreenSize();
+    searchToggle.addEventListener('click', toggleSearch);
```

Comment:
on click, search.svg spin, check css transformation of svg's

Reply:
Disabling the invocation of buttonloader as we no longer require iy. Not deleting the js file for now